### PR TITLE
update(postgresql): remove the superuser extensions list

### PR DESCRIPTION
**Pull request by wojcik-dorota**
https://aiven.atlassian.net/browse/BF-2914

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

File: docs/products/postgresql/reference/list-of-extensions.md [hunk 1]
```diff
@@ -16,12 +16,6 @@ Some extensions:
 To list the extensions and their details, such as extension version numbers,
 run `select * from pg_available_extensions` in your Aiven for PostgreSQL server.
 
-:::note
-Not all extensions listed in `pg_available_extensions` can be installed. See
-[Superuser-only extensions](/docs/products/postgresql/reference/list-of-extensions#avn_superuser_only_extensions)
-for more information.
-:::
```
Comment (tkren) [inline docs/products/postgresql/reference/list-of-extensions.md:23]: I think we can leave this note as:

````markdown
:::note
Not all extensions listed in `pg_available_extensions` can be installed. In general,
superuser-only extensions are not available to be installed by users, and can be listed with
```
SELECT name, version FROM pg_available_extension_versions WHERE superuser AND NOT trusted 
```
:::
````

Comment (wojcik-dorota) [reply docs/products/postgresql/reference/list-of-extensions.md:23]: Using pgextwlist, the users might need ops help as I understand, and we only list as "available" the extensions that the users can install themselves with on support.

Comment (tkren) [reply docs/products/postgresql/reference/list-of-extensions.md]: I guess we can list the exceptions with
```
SELECT *
FROM pg_available_extension_versions
WHERE name = ANY(string_to_array(current_setting('extwlist.extensions')::text, ','));
```

**Review by tkren — APPROVED** [review]

**Review by staceysalamon-aiven — APPROVED** [review]
LGTM

**Pull request merged**